sound/soc/sof/pm.c

```diff
@@ -221,7 +221,7 @@ static int sof_suspend(struct device *dev, bool runtime_suspend)
 	sof_tear_down_pipelines(sdev, false);
 
 	/* release trace */
-	snd_sof_release_trace(sdev);
+	snd_sof_release_trace(sdev, 0);
```

Comment:
the commit message needs to be expanded to bit to explain how you're using teh new state. It is very hard to follow what you're diong here without any explanation. And why 0? did you mean false?